overrides/scripts/Earlygame.zs

```diff
@@ -374,6 +374,49 @@ cutter.recipeBuilder()
 	.EUt(30)
 	.buildAndRegister();
 
+//Make the CPU recipes not require cleanroom
+// CPU disable cleanroom
+<recipemap:cutter>.findRecipe(120, [<metaitem:wafer.central_processing_unit>], [<liquid:lubricant> * 84]).remove();
+<recipemap:cutter>.findRecipe(120, [<metaitem:wafer.central_processing_unit>], [<liquid:distilled_water> * 253]).remove();
+<recipemap:cutter>.findRecipe(120, [<metaitem:wafer.central_processing_unit>], [<liquid:water> * 337]).remove();
+
+//CPU circuit redo
+<recipemap:cutter>.recipeBuilder()
```

Comment:
CPU requiring Cleanroom is getting fixed by CEu next update (GregTechCEu/GregTech#1176) so there is no need for this